si.proto

```diff
@@ -222,6 +220,9 @@ message AllocationAsk {
   bool Originator = 11;
   // The preemption policy for this ask
   PreemptionPolicy preemptionPolicy = 12;
+
```

Comment:
Maybe we can add comments to explain why we entirely delete this field.

Reply:
We generally don't document the removals (other than to mark them 'reserved' -- this is essentially the way things are declared obsolete).